File: learning_ws/src/learning_node_py/learning_node_py/hello_node.py
import rclpy
from rclpy.node import Node
import time

def main(args=None):
    rclpy.init(args=args)
    node = Node("node_hello_node")
    
    while rclpy.ok():
        node.get_logger().info("Hello Node")
        time.sleep(0.5)
    
    node.destroy_node()
    rclpy.shutdown()

File: learning_ws/src/learning_node_cpp/src/hello_node.cpp
#include "rclcpp/rclcpp.hpp"

class HelloNode : public rclcpp::Node
{
    public:
        HelloNode()
        : Node("node_hello_node")
        {
            while (rclcpp::ok())
            {
                RCLCPP_INFO(this->get_logger(), "Hello World");
                sleep(1);
            }
        }
};

int main(int argc, char * argv[])
{
    rclcpp::init(argc, argv);
    rclcpp::spin(std::make_shared<HelloNode>());
    rclcpp::shutdown();

    return 0;
}